src/components/auth/Login.js

```diff
@@ -11,43 +11,26 @@ import Spinner from 'react-bootstrap/Spinner';
 import loginAmico from '../assets/image/Login-amico.png';
 
 function LoginForm({ authLogin, loading, error, errResponse, token, history }) {
-  // const [values, setValues] = useState({
-  //   email: '',
-  //   password: '',
-  // });
-
   const dispatch = useDispatch();
-  // const [emailError, setEmailError] = useState(false);
-  // const handleSubmit = e => {
-  //   e.preventDefault();
-  //   authLogin(values);
-  // };
+
+  const resetAuthState = useCallback(() => {
+    if (errResponse) {
+      dispatch({ type: 'AUTH_RESET' });
+    }
+  }, [dispatch, errResponse]);
 
   useEffect(() => {
-    dispatch({ type: 'AUTH_RESET' });
-  }, [dispatch]);
+    resetAuthState();
+  }, []);
```

Comment:
Codacy found an issue: [React Hook useEffect has a missing dependency: 'resetAuthState'. Either include it or remove the dependency array.](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5730659)